### ISSUE TITLE
Version 4.2.59.0 external plugin does not work

### ISSUE THREAD

**Issue by ragebear00** [dnGrep/dnGrep#1175]
the apache tika does not work under the new version. The other plugin refer to  "{outputFile}" does not work

**Comment by doug24** [comment]
You said "{outputFile}" does not work?  What is not working? Do you get an error? Is the file empty/missing? Is the file invalid?

In my testing it does work with {outputFile} -- Here is my working Tika configuration:

```
Name=Tika
Application=cmd.exe
Arguments=/c tika.bat "{inputFile}" "{outputFile}"
WorkingDirectory=C:\Users\user\Apps\apache-tika
Extensions=odt, ods, odp, odf, epub, doc, htm, html, msg
```
where tika.bat is 
```
java.exe -jar tika-app-3.0.0-BETA.jar --text %1 > %2 
```

**Comment by doug24** [comment]
OK, I see the problem is when the Tika plug-in is **not** using {outputFile}, and reading standard input to get the extracted text.  There is a problem getting the encoding of the standard input stream.  I will fix that.

**Comment by ragebear00** [comment]
test again. Confirmed, not caused by {outputFile}, but the config without {outPutfile}.

**Comment by doug24** [comment]
Fixed in [4.2.84](https://github.com/dnGrep/dnGrep/releases/tag/v4.2.84.0)

**Comment by ragebear00** [comment]
test also, fixed
